Emby.ApiClient/model/dto/BaseItemDto.swift

```diff
@@ -372,6 +372,115 @@ public class BaseItemDto : JSONSerializable {
     public var currentProgram: BaseItemDto?
     
     public required init?(jSON: JSON_Object) {
-        fatalError("init(jSON:) has not been implemented: \(jSON)")
+        if  let id = jSON["Id"] as? String {
+            self.id = id
+            
+            // MOVIE START
+            self.name = jSON["Name"] as? String
```

Comment:
@LukePulverenti thanks. I'll follow the conversation there and if we decide to change our approach we can work to refactor this.